guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -161,7 +163,7 @@ function&nbsp;Polygon()&nbsp;{};
 Polygon.prototype.getSides&nbsp;=&nbsp;function()&nbsp;{};
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-implements">@implements</h4>
+#### @implements
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-implements}` to preserve the custom id.